examples/length/length.description

```diff
@@ -1,2 +1,2 @@
-// Ballerina supports the built-in method `length()`  that returns the length/size of the given value.
-// `length()` function is supported for `string`, `array`, `map`, `record`, `table`, `json`, `xml` and `tuple` types.
+// Ballerina provides a built-in `length()` function that returns the length/size of the given value.
```

Comment:
IINM we use method for functions that are attached to something. Shall we leave this as method? The spec says builtin method too.
```suggestion
// Ballerina provides a built-in method `length()` that returns the length/size of the given value.
```